js/lib.js

```diff
@@ -0,0 +1,21 @@
+function validateForm() {
```

Comment:
put this into `bbcode.js` - we'll have to split up / rename bbcode.js but `lib.js` doesn't make sense for a file that only contains the validateForm-Function.